tests/acceptance/features/webUILogin/login.feature

```diff
@@ -12,6 +12,5 @@ Feature: login users
 		When the user clicks the authenticate button
 		And the user logs in with username "admin" and password "admin" using the webUI
 		And the user authorizes access to phoenix
-		And the user browses to the files page
```

Comment:
This line is interesting because it's not mentioned in the `loginContext.js`, if I re-insert it, the race condition re-appears. So I'm fine with removing it.